book/src/ephemeral-local-volumes.md

```diff
@@ -77,6 +77,11 @@ A CSI driver is suitable for CSI ephemeral inline volumes if:
 - ephemeral inline volumes have to be supported on Kubernetes clusters
   which do not support generic ephemeral volumes
 
+A CSI driver is not suitable for CSI ephemeral inline volumes when:
+- provisioning is not local to the node
```

Comment:
I think we didn't want to necessary limit the feature to only local things as long as it could handle proper pod authentication.

Reply:
Can you give an example here?

My first thought was to say something like "provisioning is done in the controller and would need to be replicated in nodePublish", but that felt too functional. I think that's roughly the idea, though: if adding csi ephemeral support would require moving/duplicating provisioning code from the controller to the node then there's a bad smell.

But it would be good to be more precise I think.

Reply:
There could be some driver that stores per-Pod information in a backend, perhaps like a secret manager.

Agree that if you're driver is already supporting the normal PV method, and you would have to duplicate it here, that's not desirable.